components/services/bat_ads/public/cpp/ads_client_mojo_bridge.cc

```diff
@@ -58,6 +58,22 @@ void AdsClientMojoBridge::IsEnabled(IsEnabledCallback callback) {
   std::move(callback).Run(ads_client_->IsEnabled());
 }
 
+bool AdsClientMojoBridge::ShouldShowPublisherAdsOnPariticipatingSites(
+    bool* is_enabled) {
+  if (!is_enabled) {
```

Comment:
nit: while valid - it doesn't read 'valid' ... perhaps just DCHECK `is_enabled` here?

Reply:
or explicit `is_enabled == nullptr`

Reply:
Changed to `DCHECK` to match Google Style Guide